data/videos.yml

```diff
@@ -95,7 +95,7 @@
   author: dmarrazzo
   date: 2020-09-02
   tags:
-    - task assigning
+    - task assignment
```

Comment:
Consistency: the module's name is "task assigning", see https://github.com/kiegroup/kogito-apps/tree/main/task-assigning"

Reply:
Good point!
In the website, all other tags use "task assignment". So either the website changes or the quickstart module name changes.

Reply:
I 'll do this in a separate PR.

Reply:
Created this jira: https://issues.redhat.com/browse/PLANNER-2499

Note that "task assignment" is part of the landing page URL. Changing that is SEO sensitive (permanent 301 is a workaround)
The task assigning example should become a quickstarts in time, and then we can call it task assignment?